pandas/tests/util/test_assert_frame_equal.py

```diff
@@ -1,3 +1,5 @@
+from unittest.mock import patch
```

Comment:
i think elsewhere we use pytest's monkeypatch fixture, does that not provide the call_count check?

Reply:
I did not find this functionality there.